tasks/gitlab_helpers.py

```diff
@@ -281,3 +283,57 @@ def print_entry_points(ctx):
     print(len(entry_points), 'entry points:')
     for entry_point, config in entry_points.items():
         print(f'- {color_message(entry_point, Color.BOLD)} ({len(config)} components)')
+
+
+def compute_gitlab_ci_config_diff(ctx, before: str, after: str):
+    """
+    Computes the full configs and the diff between two git references.
+    The base commit where after is compared to is the LCA commit between before and after.
```

Comment:
```suggestion
    The "after reference" is compared to the Lowest Common Ancestor (LCA) commit of "before reference" and "after reference".
```
Don't hesitate to rephrase:
- I think it's nice to put the meaning of an acronym before using it
- using quotes help reading the sentence